packages/viewer/src/utils/components/TextAreaInput.vue

```diff
@@ -1,24 +1,11 @@
 <script setup lang="ts">
-import { useTemplateRef } from 'vue'
+import { toRefs, useTemplateRef } from 'vue'
 import { useI18n } from 'vue-i18n'
 
 import { useComponentUniqueId } from '@/utils/composables/useComponentUniqueId'
 import { useFieldValidation } from '@/utils/composables/useFieldValidation'
 
-const {
-    label = '',
-    description = '',
-    disabled = false,
-    placeholder = '',
-    required = false,
-    validMarker = undefined,
-    validMessage = '',
-    invalidMarker = undefined,
-    invalidMessage = '',
-    activateValidation = false,
-    validate = undefined,
-    dataCy = '',
-} = defineProps<{
+const props = withDefaults(defineProps<{
```

Comment:
`validMarker` and `invalidMarker` are already flagged with `?` (optional), I don't think it's required to set a defautl value to them anyway.

If it's the case because the TS type-checking complains otherwise, I'd not use withDefaults, but do a two phase deconstruct.
```TS
const props = defineProps<...>()

const { propsNeedingDefault = undefined } = props

// don't do a full `toRefs` on all the props (it will wrap some that we don't require) but specifically ref some
const label = toRef(props, 'label' /* eventual default value here as 3rd arg */)
// ...
```